package gameevent;

import "box2d.proto";


message EntityGameEvent
{
	enum EntityGameEventType
	{
		CREATE = 0;
        RETRIEVE = 1;
		UPDATE = 2;
        DESTROY = 3;
	}

	required EntityGameEventType type = 1;
    required uint64 entityTag = 2;

	optional string UUID = 3;
    optional box2d.PbBody body = 4;
}
